beacon_node/beacon_chain/src/beacon_chain.rs

```diff
@@ -83,30 +83,31 @@ impl BlockProcessingOutcome {
     }
 }
 
-pub struct BeaconChain<T: ClientDB + Sized, U: SlotClock, F: ForkChoice> {
+pub struct BeaconChain<T: ClientDB + Sized, U: SlotClock, F: ForkChoice, B: EthSpec> {
```

Comment:
`B` for "Beacon"?

Reply:
Yeah, the type used to be call `BeaconStateTypes`. It doesn't really make sense now, would be better as `E`.

I'll see if I can find all of these and fix them.

Reply:
I don't know if it's worth spending time on, `B` is fine really